meta/main.yml

```diff
@@ -1,20 +1,11 @@
 ---
 galaxy_info:
-  author: Paul Durivage
-  description: Docker on Ubuntu greater than 12.04
+  author: Location Labs
+  description: Docker on Ubuntu >= 12.04
   license: Apache v2.0
-  min_ansible_version: 1.2
+  min_ansible_version: 1.9.6
   platforms:
     - name: Ubuntu
-      versions:
```

Comment:
I hate the damn code names. Does ansible let you put coments at the end of lines, so that we could add numbers, like:

```
  - precise  # 12.04
```